crates/solidity/inputs/schema/grammar/topics/01-file-structure/07-keywords.yml

```diff
@@ -0,0 +1,710 @@
+- name: "AbicoderKeyword"
```

Comment:
I was planning on starting on these keywords refactoring sometime after alpha. I know it must have been a significant amount of work. Thank you for taking this on!

The new keywords topic looks great!